enterprise/user-code-namespaces/README.md

```diff
@@ -96,4 +96,42 @@ mvn clean compile exec:java -Dexec.mainClass=com.hazelcast.namespaces.dyamicconf
 - Gets the hazelcast config from the client instance and adds the `UserCodeNamespaceConfig` to the `Config`.
 
 
+Observe the value for key is incremented by 1 each time the client executes.
+
+-----
+
+### 4. Dynamically configured namespace with a JAR containing a single class.
+
+
+Run Member:
+
+```
+cd user-code-namespaces-member
+mvn clean compile exec:java -Dexec.mainClass=com.hazelcast.namespace.dynamic.Member
+```
+
+**Member code walkthrough:**
+
+- Enables user code namespaces feature on config.
+- Creates a new `UserCodeNamespaceConfig` named `ucn1` with no resources.
+*Note - this config must exist on the running instance before the client can add to it.*
```

Comment:
This is not correct - clients can dynamically create a new Namespace instance at runtime (provided they have the required permissions)

Reply:
I copied this warning from `3. Dynamically configured namespace with a single class.` section.  

@gareth-johnston  wrote that note, should we also remove it?

Reply:
Yes I think so, unless @gareth-johnston I'm missing something here?

Reply:
I think the point this was trying to make, admittedly not clearly enough, is the following: 

We start up a member which statically creates a map, the map references an empty namespace `ucn1` that has just been created on the member.

We run a client that wants to configure `ucn1` whilst member is running. This is where this note comes in ->  *Note - this config must exist on the running instance before the client can add to it.*

The point being that the IMap which is statically configured references a namespace - so in this case it must be pre-configured before the client can do a replace call.

Does that make sense? I can see how it is possibly not clear enough

Reply:
@mehmettokgoz if you're able to apply this change, we can get this PR merged 👍